add link to cms instructions in admin header Closes #56

scottalexandra opened the pull request:
added CMS Instuctions link on the admin navbar
added a test to admin_dashboard_spec
All tests passing

scottalexandra committed:
added link to cms instructions in admin header

Lydias303 commented on config/routes.rb:
```diff
@@ -9,6 +9,7 @@
+    get "/dashboard/cms-instructions", to: "dashboard#cms_instructions", as: "cms_instructions"
     resources :companies, except: [:create] do
       resources :notes, only: [:index, :create, :destroy]
     end
```
Do we want this to be its own page?

scottalexandra committed:
fixed merge conflicts

scottalexandra committed:
added and styled cms instructions

scottalexandra commented:
Added cms_instructions action to admin dashboard controller
added cms_instructions view in admin/dashboard folder
added _cms_instructions partial
styled cms instructions page 
all tests passing!

scottalexandra committed:
fixed typo

oorion commented on config/routes.rb:
```diff
@@ -12,6 +12,7 @@
 
   namespace :admin do
     resources :dashboard, only: [:index]
+    get "/dashboard/cms-instructions", to: "dashboard#cms_instructions", as: "cms_instructions"
```
Looks great! :sparkle:

oorion commented on spec/features/admin_dashboard_spec.rb:
```diff
@@ -14,6 +14,17 @@ def login
       click_link_or_button('Log in')
     end
 
+    it "can click a link to view cms instructions" do
```
Nice test!

bhargavisatpathy commented on spec/features/admin_dashboard_spec.rb:
```diff
@@ -14,6 +14,17 @@ def login
+      within('.cms-instructions-header') do
+        expect(page).to have_content('Content Management System(CMS) Instructions')
+      end
+    end
```
Looks good to me too :+1:
